histomicstk/Membraneness.py

```diff
@@ -4,10 +4,10 @@
 from Hessian import Hessian
```

Comment:
Let's discuss tomorrow - at some point we wondered whether or not we would stick with skimage. At this point it is pretty embedded in the code.